GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1643,25 +1632,30 @@ private void FileToolStripMenuItemDropDownOpening(object sender, EventArgs e)
 
         private void ChangeWorkingDir(string path)
         {
-            GitModule module = new GitModule(path);
+            var repository = _repositoryHistory.FirstOrDefault(r => r.Path == path);
+            if (repository == null)
```

Comment:
Why do we need to check this?

Reply:
Good catch, thanks. It is not needed.
I thought it was an existing code, but it appears I introduced few commits before this.